src/helpers/wallet.helper.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { HathorWallet, walletUtils, Connection, errors } from '@hathor/wallet-lib';
+import config from '../config';
+
+export class WalletStartError extends Error {}
+
+export function getReadonlyWallet({
+  xpub,
+  multisigData = null,
+  preCalculatedAddresses = [],
+}) {
+  const connection = new Connection({
+    network: config.network,
+    servers: [config.server],
+    connectionTimeout: config.connectionTimeout,
+  });
+
+  // Previous versions of the lib would have password and pin default as '123'
+  // We currently need something to be defined, otherwise we get an error when starting the wallet
+  const walletConfig = {
+    xpub,
+    connection,
+    password: '123',
+    pinCode: '123',
```

Comment:
Same as comment below, this seems so weird to me, can't we use an empty string here if it's a required param?

Reply:
Tracking this on #246